server/test/server_integration_test.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/dalc/config"
+	"github.com/celestiaorg/dalc/proto/dalc"
+	"github.com/celestiaorg/dalc/proto/optimint"
+	"github.com/celestiaorg/dalc/server"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spm/cosmoscmd"
+	"google.golang.org/grpc"
+)
+
+var (
+	dalcClient dalc.DALCServiceClient
+)
+
+// TestIntegration is only meant to run when connected to a celestia full node
+// and a celestia-node
+func TestIntegration(t *testing.T) {
+	t.Skip("test requires connection to a full node and celestia-node")
```

Comment:
skipping this test, as it requires running a celestia-app network, connecting a celestia-node full node, and then properly configuring the dalc's celestia-node light as well. It can however be used to test things locally.